docs/docs/getting-started/integration.md

```diff
@@ -22,9 +22,17 @@ var myChart = new Chart(ctx, {...});
 
 ## Bundlers (Webpack, Rollup, etc.)
 
-Chart.js 3 is tree-shakeable, so it is necessary to import and register the controllers, elements, scales and plugins you are going to use.
+To use Chart.js, a simple import is all that is required.
+
+```javascript
+import Chart from 'chart.js';
+var myChart = new Chart(ctx, {...});
+```
+
+### Tree Shaking
+
+Chart.js 3 has optional tree shaking that is enabled via the `chart.js/tree` package. To use this, it is necessary to import and register the controllers, elements, scales and plugins you wish to use. The example below shows all of the available imports and how to register them.
```

Comment:
I think users expect the default behavior (i.e. `import {...} from 'chart.js'`) to be tree-shakable, which may be the most common use case. It's also the natural method to import from Chart.js and it may be better to keep it that way. As a plugin developers, I would also expect the following snippet to only bring `LineElement` and not trigger any auto-registration:

```js
import { LineElement } from 'chart.js';

export class CurveElement extends LineElement {
  // ...
}
```

It may be better to have a special case for auto-registering instead (e.g. `chart.js/auto` or `chart.js/all`).